### PR TITLE
Tentative

### PR DESCRIPTION
**Pull request by alias01**

### CODE REVIEW

File: src/components/loginModalForm.vue [hunk 1]
```diff
@@ -98,6 +100,7 @@ export default {
           Password: this.loginPassword,
         });
         var token = resp.token || "";
+        console.log(token)
```
Comment (arrogant-ant) [inline src/components/loginModalForm.vue:103]: we can remove this

File: src/pages/srp/components/BookMsg.vue [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+          <p>Warning</p>
+          <button
+            v-on:click="onCancel"
+            class="delete"
+            aria-label="delete"
+          ></button>
+        </div>
+        <div class="message-body">
+            <p>Sorry for the inconvenience!! </p>
+          <p>It seems Spot is already filled...</p>
```
Comment (arrogant-ant) [inline src/pages/srp/components/BookMsg.vue:31]: we can replace this with resMsg.DisplayMsg

File: src/views/Srp.vue [hunk 3]
```diff
@@ -1,23 +1,89 @@
...
+            Name: book.name,
+            Mobile: book.mno,
+            EmailID:book.email,
+            VehicleNO: "KA01JE3635",
+          },
+        }),
+      });
+      const data = await res.json();
+      // const check =JSON.parse(data)
+      console.log(data);
```
Comment (arrogant-ant) [inline src/views/Srp.vue:70]: please remove log too

File: src/views/Srp.vue [hunk 4]
```diff
@@ -1,23 +1,89 @@
...
+	  this.resMsg = data;
+	  this.resMsgflag = !this.resMsgflag
+	  
+    },
+    onBookFormtoggle(index) {
+      this.showBookForm = !this.showBookForm;
+      // console.log(index)
```
Comment (arrogant-ant) [inline src/views/Srp.vue:77]: please remove unwanted comment

**Review by arrogant-ant — CHANGES_REQUESTED** [review]

File: src/components/loginModalForm.vue (hunk 1 above)
Comment (alias01) [reply src/components/loginModalForm.vue]: done

File: src/pages/srp/components/BookMsg.vue (hunk 2 above)
Comment (alias01) [reply src/pages/srp/components/BookMsg.vue]: done

File: src/views/Srp.vue (hunk 3 above)
Comment (alias01) [reply src/views/Srp.vue]: done

File: src/views/Srp.vue (hunk 4 above)
Comment (alias01) [reply src/views/Srp.vue]: done

**Review by arrogant-ant — APPROVED** [review]

**Pull request merged**